ipp_pd_async.py

```diff
@@ -0,0 +1,82 @@
+import dbus
+import dbus.mainloop.glib
+import gi
+from gi.repository import GLib
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
+import avahi
```

Comment:
Some distros currently do not ship Avahi Python module - can you catch the exception, show the error dialog if someone clicks on "Discover Services" about "Missing Avahi module - we could not discover mDNS services." and return back to the main window?

Reply:
Sure , I'll do require changes